datalad/support/gitrepo.py

```diff
@@ -1087,6 +1087,7 @@ def fetch(self, remote=None, refspec=None, progress=None, all_=False,
                 # TODO: progress +kwargs
 
         # TODO: fetch returns a list of FetchInfo instances. Make use of it.
+        return fi_list
```

Comment:
docstring still say "Nothing yet." for Returns...

could you 'solidify' what is returned in the test, i.e. I bet there is a test already and you would probably need just to check return value